Lab 6

d3nik opened the pull request.

d3nik committed:
Patch 6.1

– added footer email form;
– added inputs for modal window;

d3nik committed:
Patch 6.1.1 (hotfix)

– deleted content from .custom-checkbox;

d3nik commented:
у мене чомусь іконка не появлялась, коли я вставляв спрайт в background-image або в content через url

d3nik committed:
Patch 6.1.2 (hotfix)

– fixed custom-checkbox tick icon;

vozniuk-lntu reviewed (approved):
1. в інпута при фокусі просто колір бордера міняй, не треба аутлайн. Іконки теж мають міняти колір
2. При кліці на Погоджуюся... маєь натискатись чекбокс. Відцентурй текст. Колір тексту.
3. .footer .container там геп 94пх, а з .footer-right-side боковий падніг прибрати
4. транзішени не всюди є

d3nik committed:
Patch 6.1.3 (hotfix)

– fixed outline to border in modal inputs;
– text in checkbox now clickable, changed color and moved to center;
– fixed transitions;
– fixed footer gaps to 94px;
